RockWeb/Webhooks/Lava.ashx

```diff
@@ -109,6 +109,11 @@ public class Lava : IHttpHandler
                 string apiMethod = api.GetAttributeValue( "Method" );
                 List<string> variables = new List<string>();
 
+                if ( !apiUrl.StartsWith( "/" ) && !apiUrl.StartsWith( "^" ) )
```

Comment:
@cabal95 Wouldn't this be better to add after the check below for apiUrl being null or empty?

Reply:
Yes... Yes i would. :) Done.